docs/main/docs/components/become-an-executor.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 2
+---
+
+# How to Become an Executor
+
+This documentation covers how to become an Executor on the t3rn network. It assumes that you have accounts with the correct funds for the desired blockchains, and that you have Substrate installed, which is needed for generating keys with subkey; we will share the resources to complete these steps.
```

Comment:
I wouldn't say `subkey` is strict requirement. One can already have keys or generate them with other tool. 
Maybe rephrase to keys are needed and here we use `subkey` to generate them...